spec/units/database_helper.rb

```diff
@@ -1,8 +1,12 @@
+# frozen_string_literal: true
+
 require 'pg'
 
-def truncate_table(table)
+p 'Setting up test database...'
+
+def truncate_test_database
   connection = PG.connect(dbname: 'mmbb_test')
-  connection.query("delete FROM #{table}")
+  connection.exec('TRUNCATE listing;')
```

Comment:
Not passing in a table as an argument but that's fine, we'll just need to remember to add new tables in after listing if we add any more to our database